launch_testing/pytest.ini

```diff
@@ -3,3 +3,4 @@
 testpaths = test
 # Add arguments for launch tests
 addopts = --launch-args dut_arg:=test
+junit_family=xunit1
```

Comment:
This is not compatible with ci.ros2.org: https://github.com/ros2/ci/blob/a7c41fbce3f1acee2a98a0e2865a8c9a0fb0db8a/ros2_batch_job/__main__.py#L381

Reply:
Ah, thanks for the feeback.  Will set to xunit2.

Reply:
CI already sets the flag automatically. So why do you want to set it explicitly (which would be the only package which does that)?

Reply:
Because otherwise there's a warning from CI:

```
  Warning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
  Add 'junit_family=xunit1' to your pytest.ini file to keep the current format in future versions of pytest and silence this warning.
```

I *think* this needs a custom pytest.ini to add options, but I really don't know.  @ivanpauno @hidmic any thoughts here?

Reply:
I think that if the warning is silenced by explicitly specifying the `junit_familiy` here and CI passes, it's ok this approach (besides that CI already sets the junit family).

Reply:
> I think this needs a custom pytest.ini to add options

Correct. 

> Because otherwise there's a warning from CI:

So you think the existence of `pytest.ini` itself triggers the warning?

Reply:
Can you please reference the build where you get this warning. On ci.ros2.org this should happened because of ros2/ci#406.

Reply:
> I think that if the warning is silenced by explicitly specifying the `junit_familiy` here and CI passes, it's ok this approach (besides that CI already sets the junit family).

The question is *why* it is necessary since the CI logic tries to workaround this already. And if it is necessary I would assume it is the same for *all* packages and then should also be addressed for all.

Reply:
Oh, right.  This doesn't happen in CI because of the override.  I only saw it *locally*, which is why we need it.

Reply:
>  I only saw it locally, which is why we need it.

Then please apply it in a separate pull request to *all* packages.

Reply:
> Then please apply it in a separate pull request to _all_ packages.

I really don't see why.  This improves it for this package; adding it to other packages to fix it there is orthogonal.